ts/components/cie/CieNotSupported.tsx

```diff
@@ -0,0 +1,71 @@
+import { Body, List, ListItem, Text, View } from "native-base";
+import * as React from "react";
+import { Platform } from "react-native";
+import I18n from "../../i18n";
+import customVariables from "../../theme/variables";
+import IconFont from "../ui/IconFont";
+import Markdown from "../ui/Markdown";
+
+type Props = {
+  hasCieApiLevelSupport: boolean;
+  hasCieNFCFeature: boolean;
+};
+
+const CieNotSupported: React.FunctionComponent<Props> = props => {
+  return (
+    <React.Fragment>
+      <Markdown>
+        {I18n.t("authentication.landing.cie_unsupported.body")}
+      </Markdown>
+      {Platform.OS === "android" && (
+        <React.Fragment>
+          <View spacer={true} />
+          <Markdown>
+            {I18n.t("authentication.landing.cie_unsupported.android_desc")}
+          </Markdown>
+          <View spacer={true} extralarge={true} />
+          <List>
+            <ListItem>
+              <IconFont
+                name="io-tick-big"
+                size={16}
```

Comment:
Avoid magic number
Place it as const on top of file

Reply:
Fixed in 6533ccb